decidim-blogs/spec/system/explore_posts_spec.rb

```diff
@@ -45,7 +45,6 @@
       expect(page).to have_i18n_content(post.title)
       expect(page).to have_i18n_content(post.body)
       expect(page).to have_content(post.author.name)
-      expect(page).to have_content(post.created_at.day)
```

Comment:
haha.. I also suffered this, there's a fix in one of my PRs

Reply:
I reverted the change!